pyrit/datasets/orchestrators/tree_of_attacks/red_teaming_prompt_template.yaml

```
@@ -0,0 +1,17 @@
+name: tree_of_attacks_with_pruning_prompt_template
+description: Based on the paper 'Tree of Attacks Jailbreaking Black-Box LLMs Automatically' https://arxiv.org/abs/2312.02119
+harm_category: NA
+author: Anay Mehrotra, Manolis Zampetakis, Paul Kassianik, Blaine Nelson, Hyrum Anderson, Yaron Singer, Amin Karbasi

```

Nit: Format in authors listing. I see both formats but one author per line is more common that I've seen in YAML templates.
```suggestion
author: 
  - Anay Mehrotra
  - Manolis Zampetakis
  - Paul Kassianik
  - Blaine Nelson 
  - Hyrum Anderson
  - Yaron Singer
  - Amin Karbasi
```

---

#282